index.html

```diff
@@ -53,6 +55,139 @@ <h2>I am a Software Developer</h2>
                 </a>
             </li>
         </ul>  
-    </section>    
+    </section>
+    <article class="works">
+        <section class="main-project">
+            <h1 class="banner">My Recent Works</h1>
+            <img src="assets\Img Placeholder.png" alt="">
+            <h2>Multi-Post Stories</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div>
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+        </section>
+        <section class="other-projects">
```

Comment:
- I think it will be better for you to wrap your  entire `class="works"` container in a `<section>` tag  instead of an `<article>`    as this defines the **works section** of your portfolio. 👍🏽

- Likewise, I believe `<section class="main-project">` and `<section class="other-projects">` can be wrapped around `<div>` tags as they are sub-sections of your "class="works"" container. 🥂